test/new-e2e/tests/installer/package_apm_inject_test.go

```diff
@@ -301,6 +301,28 @@ func (s *packageApmInjectSuite) TestInstrument() {
 	s.assertDockerdInstrumented(injectOCIPath)
 }
 
+func (s *packageApmInjectSuite) TestPackagePinning() {
+	// Deb install using today's defaults
+	err := s.RunInstallScriptWithError(
+		"DD_APM_INSTRUMENTATION_ENABLED=all",
+		"DD_APM_INSTRUMENTATION_LIBRARIES=python:2.8.2-dev,dotnet",
```

Comment:
what's the behaviour if the version is not supported by OCI (ex: an old version from deb-rpm)?
Expected for the migration should be that the install works without error and that `is-installed` answers false so that the deb/rpm gets installed

Reply:
👍  I added this case in the e2e with the ruby library